mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/MailboxMappingFactory.java

```diff
@@ -83,6 +85,10 @@ public static XContentBuilder getMappingContent() {
             return jsonBuilder()
                 .startObject()
 
+                    .startObject(SOURCE)
+                        .field(ENABLED, false)
+                    .endObject()
```

Comment:
Then... How do we perform flags update on top of ElasticSearch? 

Do we get some tests/integration test failing?